extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -84,6 +88,9 @@ export class AmpAdApiHandler {
     this.iframe_ = iframe;
     this.is3p_ = is3p;
     this.iframe_.setAttribute('scrolling', 'no');
+    let iframeSrc = this.iframe_.getAttribute('src');
+    iframeSrc = iframeSrc.substring(0, iframeSrc.indexOf('#'));
+    this.useApi_ = endsWith(iframeSrc, '.html');
```

Comment:
I don't know how to make `baseInstance instanceof AmpA4A` work here? That way we would need to import `AmpA4A` instance to amp-ad extension. now only `amp-a4a` depend on files from `amp-ad` files, I am afraid things won't work if we create a loop here